spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/ribbon/DefaultServerIntrospector.java

```diff
@@ -26,12 +26,17 @@
  */
 public class DefaultServerIntrospector implements ServerIntrospector {
 	@Override
-	public boolean isSecure(Server server) {
+	public boolean secureAvailable(Server server) {
```

Comment:
@spencergibb I was hesitant about that, too. My thinking was that isSecure, when it represents securePortEnabled in the case of the EurekaServerIntrospector, is an actively misleading security-related method. Knowing that the interface is used in only two places and that I'd need to modify it anyway with the additional insecure method, I erred on the side of code clarity. That said, should we stick with this or refactor for consistency to isSecure and isInsecure?